Common/utils.ts

```diff
@@ -320,4 +321,26 @@ export function getSeasonWithLeaguePage(seasons: SeasonModel[], seasonId?: strin
     return null;
   }
   return seasons.find(season => season.id.toString() === seasonId.toString() && season.googleSheetParserType !== null && season.googleSheetId !== null);
-}
\ No newline at end of file
+}
+
+export function MakeId(length: number) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  let counter = 0;
+  while (counter < length) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    counter += 1;
+  }
+  return result;
+}
```

Comment:
Im pretty sure theres a better way to do this, but ig this works for now. IIRC normally people filter out bad words etc.